usaspending_api/search/migrations/0048_replace_pac_pan_columns_with_program_activities.py

```diff
@@ -0,0 +1,52 @@
+# Generated by Django 3.2.25 on 2024-12-20 01:14
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('search', '0047_add_program_activity_to_search'),
+    ]
+
+    operations = [
```

Comment:
I believe this is the case, but can you confirm that these fields are not currently being used anywhere?

Reply:
They are not being used anywhere else. Thankfully, these two columns haven't been around long enough to be used anywhere else.